### PR TITLE
EN: New help file: Sharing a bib file

### PR DESCRIPTION
**Pull request by mlep**
For review.

### CODE REVIEW

File: en/SharedBibFile.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# Sharing a Bib(La)TeX Database
+
+JabRef allows to share both Bib(La)TeX database and [SQL database](SQLDatabase).
+
+When sharing a Bib(La)TeX database, JabRef automatically recognizes a change in the `bib` file on disk and notifies the user of it.
+This works well on network drives.
+
+*Note:* the use of a version control system (SVN, git, etc.) is recommended as this will allow for reverting changes.
+
+To make the sharing of a Bib(La)TeX database easier, it is recommanded to set specific database properties.
```
Comment (oscargus) [inline en/SharedBibFile.md]: recommended (e instead of a)

File: en/SharedBibFile.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+In the menu **File -> database properties**:
+- Select `UTF-8` as encoding.
+- Define a `General file directory`, which will be used to store shared PDF (and other) files.
+- Define a sort order (`year`, `author`, `title` is recommended).
```
Comment (oscargus) [inline en/SharedBibFile.md]: Is this really helping? I would expect "original order" to produce much less conflicts, but I have no actual experience.

**Comment by oscargus** [comment]
Looks good! I know from experience that using different line endings will confuse git/svn. Is there a way to handle this in JabRef (without setting the preferences to use the same line ending for every computer, which I guess is not really a good idea. Maybe we should add a database property for this as well?

I also know that you can tell git/svn to normalize this, but I do not really know how. It would be great if someone with that knowledge can add the information here as a service to the users.

File: en/SharedBibFile.md (hunk 1 above)
Comment (mlep) [inline en/SharedBibFile.md]: Corrected.

**Comment by mlep** [comment]
This was (is?) an issue: <https://sourceforge.net/p/jabref/feature-requests/857/>

**Comment by oscargus** [comment]
You can set it now, but if I recall correctly (on the phone now) it is in the preferences, not the database properties.

**Comment by oscargus** [comment]
I checked it and it is set from the preferences. Should really be on a file by file basis. Will add an issue tomorrow.

**Pull request merged**